rest/main.go

```diff
@@ -50,7 +50,7 @@ func serverMain(ctx context.Context, osArgs []string) error {
 	metricsInterfaceAuthFlag := fs.Bool("api.metrics_interface_authentication", true, "")
 
 	allowInsecureServerConnections := fs.Bool("bootstrap.allow_insecure_server_connections", false, "")
-	allowInsecureTLSConnections := fs.Bool("bootstrap.allow_insecure_tls_connections", false, "")
+	allowInsecureTLSConnections := fs.Bool("api.https.allow_insecure_tls_connections", false, "")
```

Comment:
There's another in config_legacy.go in ParseCommandLine that needs to change

Reply:
Thanks, glad you spotted that as I forgot about that.